### PR TITLE
Remove obsolete firebase deployment.

### PR DESCRIPTION
**Pull request by Hyperkid123**
Removes firebase dependencies.

### CODE REVIEW

File: packages/react-renderer-demo/firebaseFunctions.js [hunk 1]
```diff
@@ -1,96 +0,0 @@
...
-      nextjsServer.serveStatic(req, res, filePath);
-    } else {
-      nextjsHandle(req, res, parsedUrl);
-    }
-  });
-});
-
-const github = require('@actions/github');
-
-exports.sendComment = functions.https.onRequest(async (request, response) => {
```
Comment (rvsia) [inline packages/react-renderer-demo/firebaseFunctions.js:66]: @Hyperkid123 We should keep this part (and migrate it to somewhere else) as it handles the bot.

Is it possible to migrate it to vercel as well? 🤔

Comment (rvsia) [reply packages/react-renderer-demo/firebaseFunctions.js:66]: https://vercel.com/docs/concepts/functions/serverless-functions

Comment (Hyperkid123) [reply packages/react-renderer-demo/firebaseFunctions.js:66]: I can't do it now I am on meetings whole week. If you create an issue I can work on it later. I just want the build to stop failing.

Comment (rvsia) [reply packages/react-renderer-demo/firebaseFunctions.js:66]: Yeah, actually we can remove it - as long we won't remove anything from the firebase itself, it should be fine.

**Review by rvsia — APPROVED** [review]

**Comment by Hyperkid123** [comment]
@rvsia its running now. There was still a reference to the deploy job in circle ci config.

**Pull request merged**